i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+from i6_models.config import ModelConfiguration
+from i6_models.parts.conformer.norm import LayerNorm
+from typing import Callable, Union, Any, Type
+
+
+@dataclass
+class ConformerConvolutionV1Config(ModelConfiguration):
+    channels: int
+    """number of channels for conv layers"""
+    kernel_size: int
+    """kernel size of conv layers"""
+    dropout: float
+    """dropout probability"""
+    activation: Union[nn.Module, Callable[[torch.Tensor], torch.Tensor]]
+    """activation function applied after norm"""
+    norm: Union[nn.Module, Callable[[torch.Tensor], torch.Tensor]]
+    """Type of normalization layer"""
+
+
+class ConformerConvolutionV1(nn.Module):
+    """
+    Conformer convolution module.
+    see also: https://github.com/espnet/espnet/blob/713e784c0815ebba2053131307db5f00af5159ea/espnet/nets/pytorch_backend/conformer/convolution.py#L13
+    """
+
+    def __init__(self, model_cfg: ConformerConvolutionV1Config):
+        """
+        :param model_cfg: model configuration for this module
+        """
+        super().__init__()
+
+        channels = model_cfg.channels
+        kernel_size = model_cfg.kernel_size
+        dropout = model_cfg.dropout
+        activation = model_cfg.activation
+        norm = model_cfg.norm
```

Comment:
I would not create these locals vars here. Why not directly access `model_cfg` in the code below? That looks simpler to me.

Reply:
I think this is fine. I would prefer to see the list of model configurations in the constructor.

Reply:
ok i made it simpler.